content/en/blog/_posts/Windows-Memory-Pressure-Eviction.md

```diff
@@ -0,0 +1,85 @@
+
+For Windows, the available memory is derived from the node's global memory commit levels, which are queried through the [GetPerformanceInfo()](https://](https://learn.microsoft.com/windows/win32/api/psapi/nf-psapi-getperformanceinfo)) system call. Available memory is calculated by subtracting the node's global `CommitTotal` (the amount of memory committed by all processes on the node) from the `CommitLimit` (the total amount of virtual memory that can be committed without extending the page file(s) on the node).
+
+> Important note: The `CommitLimit` (capacity) can change dynamically based on current memory usage if Windows is configured to automatically adjust page file sizes, which is the default behavior on Windows Client machines.
+
+This approach differs from Linux, where available memory is reported based on **Working Set** memory usage.
+
+## Why Use Commit Memory on Windows?
+
+Before diving into the reasons for using commit memory on Windows for eviction signalling, it's helpful to understand the difference between **Working Set** and **Commit** memory.
+
+**Working Set** memory represents the amount of physical memory currently used by processes. It includes the pages of memory that are actively in use and exludes memory pages that have been swapped out to disk (if paging is enabled).
+
+**Commit** memory, on the other hand, represents the total amount of virtual memory that has been reserved by processes - including memory that has been swapped out to disk. When memory is "committed", it means that the system has allocated the backing storage (either physical RAM or space in the paging file(s)) to ensure that the memory will be available if needed.
+
+Using global commit levels to measure available memory on Windows was implemented for two key reasons:
+
+1. **Memory limits for containers in Pods running on Windows nodes are enforced using Commit** (not WorkingSet) **memory**. Windows memory management enforces limits on processes based on the total amount of memory committed, not just the memory in use. 
+This also means that memory limits specified for containers in Kubernetes Pods are enforced on the total amount of virtual memory committed by all processes in the container. Consistenty in how memory usage is tracked and enforced is essential for workload stability.
+
+2. **Paging (swap) is automatically enabled on Windows**, and it plays an essential role in the operating system's memory management. Disabling paging is not recommended because it can lead to system instability.
+
+## Eviction Thresholds
+
+By default, Windows nodes have a default "hard" eviction threadhold of `--eviction-hard=memory.available<500Mi`. This means that if the system's availalbe memory is less than 500 Mi the Kubelet will try and start evicting pods.
```

Comment:
```suggestion
By default, Windows nodes have a default "hard" eviction threadhold of 500Mi (kubelet setting `--eviction-hard=memory.available<500Mi>`). This means that if the system's available commit memory is less than 500 Mi the Kubelet will try and start evicting pods.
```

Reply:
this still looks off, specifically the spelling and the missing bracket

Reply:
the bracket isn't missing, that is supposed to be a less-than operator.
I have fixed the spelling mistakes and added some clarity tho